akka-sample-cluster-scala/src/main/resources/application.conf

```diff
@@ -1,8 +1,12 @@
 akka {
+  loglevel = debug
+  loggers = ["akka.event.slf4j.Slf4jLogger"]
+  logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
   actor {
     provider = cluster
   }
   remote {
+    log-remote-lifecycle-events = debug
```

Comment:
do we want to have it that verbose in samples?

Reply:
I'm not sure the remote logging is the first thing users should see or care about.
By the way, artery is enabled here so I don't think this setting has any effect.